### PR TITLE
Added some endpoints

### PR DESCRIPTION
**Pull request by huseynyusif**

### CODE REVIEW

File: src/main/java/az/unitech/bankapplication/controller/HelloController.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package az.unitech.bankapplication.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
```
Comment (peymanabdullali) [inline src/main/java/az/unitech/bankapplication/controller/HelloController.java:7]: why do you call this Hello?

File: src/main/java/az/unitech/bankapplication/controller/HelloController.java
Comment (elkhan18) [inline src/main/java/az/unitech/bankapplication/controller/HelloController.java:1]: Why do you call this hello?

File: src/main/java/az/unitech/bankapplication/controller/create/UserCreateController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/v1/users")
+public class UserCreateController {
```
Comment (peymanabdullali) [inline src/main/java/az/unitech/bankapplication/controller/create/UserCreateController.java]:
```suggestion
public class UserController {
```

File: src/main/java/az/unitech/bankapplication/entity/Account.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Entity
+@Table(name = "accounts",schema = "bank_application")
+public class Account {
```
Comment (peymanabdullali) [inline src/main/java/az/unitech/bankapplication/entity/Account.java]: 50 dəfə deyib müəllim!!!
Account - AccountEntity

Comment (elkhan18) [inline src/main/java/az/unitech/bankapplication/entity/Account.java]:
```suggestion
public class AccountEntity{
```

File: src/main/java/az/unitech/bankapplication/controller/HelloController.java (hunk 1 above)
Comment (huseynyusif) [reply src/main/java/az/unitech/bankapplication/controller/HelloController.java:7]: it is for testing

File: src/main/java/az/unitech/bankapplication/controller/create/UserCreateController.java (hunk 2 above)
Comment (huseynyusif) [reply src/main/java/az/unitech/bankapplication/controller/create/UserCreateController.java]: done

File: src/main/java/az/unitech/bankapplication/entity/Account.java (hunk 3 above)
Comment (huseynyusif) [reply src/main/java/az/unitech/bankapplication/entity/Account.java]: fixed

File: src/main/java/az/unitech/bankapplication/service/create/UserCreateService.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class UserCreateService {
+
+    private final UserRepository userRepository;
+    private final UserMapper userMapper;
```
Comment (narimansayilov) [inline src/main/java/az/unitech/bankapplication/service/create/UserCreateService.java:18]: Why did you inject directly without creating an INSTANCE?

**Pull request merged**